### PR TITLE
Update EULA link 

### LINKED ISSUE #1168: Update T&Cs to new EULA on or shortly after 7th July
### Your use case

The current Terms and Conditions will be replaced with our new End User Licensing Agreement on 7th July 2023. On or as soon as possible after that date, could we:

- Replace the existing link with the new one, which will be on https://static.element.io/legal/online-EULA.pdf
- Update the text from `Terms and conditions` to `End User Licensing Agreement (EULA)`

### Have you considered any alternatives?

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes https://github.com/vector-im/element-call/issues/1168
Based on https://github.com/vector-im/element-call/pull/1194

### CODE REVIEW

File: src/auth/RegisterPage.tsx [hunk 1]
```diff
@@ -54,8 +54,9 @@ export const RegisterPage: FC = () => {
   const [error, setError] = useState<Error>();
   const [password, setPassword] = useState("");
   const [passwordConfirmation, setPasswordConfirmation] = useState("");
-  const [privacyPolicyUrl, recaptchaKey, register] =
-    useInteractiveRegistration();
+  const interactiveRegistration = useInteractiveRegistration();
```
Comment (toger5) [inline src/auth/RegisterPage.tsx]: This is not realted to your pr but i dont like how: `useInteractiveRegistration` is returning an array and `useRegisterPasswordlessUser` returns an object.

File: src/home/UnauthenticatedView.tsx [hunk 2]
```diff
@@ -48,8 +48,9 @@ export const UnauthenticatedView: FC = () => {
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState<Error>();
   const [optInAnalytics] = useOptInAnalytics();
-  const [privacyPolicyUrl, recaptchaKey, register] =
-    useInteractiveRegistration();
+  const interactiveRegistration = useInteractiveRegistration();
+  const recaptchaKey = interactiveRegistration[1];
+  const register = interactiveRegistration[2];
```
Comment (toger5) [inline src/home/UnauthenticatedView.tsx]: maybe refacturing `useInteractiveRegistration` to use an obj would be worthit? wdyt?

**Review by toger5 — COMMENTED** [review]
Code looks good. But if I open the link with the pdf I dont see anything? Am I doing sth wrong?

 - I copied the link to the EULA and opened it in a new tab
 - I tried the netlify build and clicked on the link.

File: src/auth/useInteractiveRegistration.ts [hunk 3]
```diff
@@ -22,17 +22,17 @@ import { initClient } from "../matrix-utils";
...
 import { Config } from "../config/Config";
 
-export const useInteractiveRegistration = (): [
-  string,
-  string,
-  (
+export const useInteractiveRegistration = (): {
+  privacyPolicyUrl: string;
+  recaptchaKey: string;
+  register: (
```
Comment (toger5) [inline src/auth/useInteractiveRegistration.ts:28]: Thanks for updating this.

**Review by toger5 — APPROVED** [review]

**Comment by SimonBrandner** [comment]
Alright, this can be merged in the case SRE/EMS team isn't fast enough

**Comment by SimonBrandner** [comment]
Closing in favour of https://github.com/vector-im/element-call/pull/1199

**Pull request closed without merging**